proj4/src/test/scala/geotrellis/proj4/io/wkt/WKTTest.scala

```diff
@@ -63,4 +64,12 @@ class WKTTest extends FunSpec {
     val epsgCodeOfWKT = WKT.getEpsgStringCode("PROJCS[\"WGS 84 / Pseudo-Mercator\", GEOGCS[\"WGS 84\", DATUM[\"World Geodetic System 1984\", SPHEROID[\"WGS 84\", 6378137.0, 298.257223563, AUTHORITY[\"EPSG\",\"7030\"]], AUTHORITY[\"EPSG\",\"6326\"]], PRIMEM[\"Greenwich\", 0.0, AUTHORITY[\"EPSG\",\"8901\"]], UNIT[\"degree\", 0.017453292519943295], AXIS[\"Geodetic longitude\", EAST], AXIS[\"Geodetic latitude\", NORTH], AUTHORITY[\"EPSG\",\"4326\"]], PROJECTION[\"Popular Visualisation Pseudo Mercator\", AUTHORITY[\"EPSG\",\"1024\"]], PARAMETER[\"semi_minor\", 6378137.0], PARAMETER[\"latitude_of_origin\", 0.0], PARAMETER[\"central_meridian\", 0.0], PARAMETER[\"scale_factor\", 1.0], PARAMETER[\"false_easting\", 0.0], PARAMETER[\"false_northing\", 0.0], UNIT[\"m\", 1.0], AXIS[\"Easting\", EAST], AXIS[\"Northing\", NORTH], AUTHORITY[\"EPSG\",\"3857\"]]").get
     assert(comparisonCode == epsgCodeOfWKT)
   }
+
+  // TODO: fix proj4j, it should be identify this projection as EPSG:3857
```

Comment:
Wouldn't this code block be better as a new issue, either in this repo or in upstream proj4j if we can verify that there is in fact a bug in that code?

Reply:
https://github.com/locationtech/proj4j/issues/61